### PR TITLE
[cssom] Test that CSSStyleDeclaration.cssText returns the empty string for computed style.

### PR DESCRIPTION
**Pull request by emilio**
Test for https://github.com/w3c/csswg-drafts/issues/1033 / https://github.com/w3c/csswg-drafts/pull/4945.

### CODE REVIEW

File: css/cssom/cssstyledeclaration-csstext.html [hunk 1]
```diff
@@ -97,6 +97,10 @@
 
         }, 'invalid property does not appear');
 
+        // https://github.com/w3c/csswg-drafts/issues/1033
+        test(function() {
+            assert_equals(getComputedStyle(document.body).cssText, "");
```
Comment (upsuper) [inline css/cssom/cssstyledeclaration-csstext.html:102]: I suggest having a new element with several properties set, and then check its computed `cssText`.

Comment (emilio) [reply css/cssom/cssstyledeclaration-csstext.html]: Yeah, good call, done.

**Review by upsuper — APPROVED** [review]
LGTM

**Comment by emilio** [comment]
@jgraham / @stephenmcgruer I don't know what the error is about... What do I need to do to get this merged?

**Comment by stephenmcgruer** [comment]
(I'm afraid I don't know anything about that error, will leave that to @jgraham , but let me try closing/re-opening the PR to kick TaskCluster to run again, and maybe it'll succeed next time around.)

**Comment by emilio** [comment]
Oh well, rebased and automagically went away. I'll merge while I can :-)

**Pull request merged**